integration_tests/README.md

```diff
@@ -409,6 +409,15 @@ the SHS supported values for the config key
 With `zstd` it's easy to view / decompress event logs using the CLI `zstd -d [--stdout] <file>`
 even without the SHS webUI.
 
+### Worker Logs
+
+By default, when using xdist the integration tests will write the tests output to console and to a text file 
+that will appear under the run directory of the form 
+`integration_tests/target/run_dir-<timestamp>-xxxx/gw0_worker_logs.log`. The output format of the log and the log level  
+can be changed by modifying the file `integration_tests/src/test/resources/xdist_it_log4j.properties`. 
+
+In case of a single threaded test run i.e. `TEST_PARALLEL=1` the logs will be printed on the console only.
```

Comment:
the logs go to a file too, under target.. I can't remember exact name

Reply:
+1 let us handle the TEST_PARALLEL < 2 as well

for consistency we should use the same logic as for event logs https://github.com/NVIDIA/spark-rapids/blob/9415a5119a7cc27112041aca389da2697a75a945/integration_tests/README.md?plain=1#L395-L396

Reply:
Done